templateutil/funcs_test.go

```diff
@@ -0,0 +1,50 @@
+package templateutil
+
+import (
+	"testing"
+
+	"github.com/bitrise-io/go-utils/envutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetenv(t *testing.T) {
+	testEnvKey := "KEY_TestGetenv"
+
+	// no env set yet, empty value should be returned
+	require.Equal(t, "", Getenv(testEnvKey))
```

Comment:
pls do not use any testing framework for new tests, we want to deintroduce them.